src/ip/Address.h

```diff
@@ -298,6 +298,10 @@ class Address
      */
     bool GetHostByName(const char *s);
 
+    /// \returns an object containing the specific IP case NO_ADDR (format-neutral).
```

Comment:
Let's simplify this and avoid duplicating (rather questionable!) details:

```suggestion
    /// \returns an Address for which isNoAddr() is true
```

or

```suggestion
    /// \returns an Address with true isNoAddr()
```